src/index.ts

```diff
@@ -1,39 +1,53 @@
-import { NativeModules } from "react-native";
+import { NativeModules, Platform } from "react-native";
 
 const { RNNativeLogs } = NativeModules;
 
 export const NativeLogs = {
   currentLogsIndex: {} as { [key: string]: number },
-  
+
   /** Redirects the native logs from the console to a given file.
     * @param {string} identifier - The name of the file to write the logs to.
+    * @param {string} logLevel - [ANDROID only] The log level to redirect. Can be one of: "D" for debug, "I" for info, "W" for warn, "E" for error, "F" for fatal.
     * @returns {Promise<void>} - A promise that resolves when the logs are redirected.
   */
-  async redirectLogs(identifier: string): Promise<void> {
+  async redirectLogs(identifier: string, logLevel?: string): Promise<void> {
```

Comment:
If we can only accepts few parameters create a type called LogLevel and put only the possible values.
`type LogLevel = 'D' | 'I'`

Use it in the others place as well